lib/trento_web/openapi/v2/schema/cluster.ex

```diff
@@ -0,0 +1,175 @@
+defmodule TrentoWeb.OpenApi.V2.Schema.Cluster do
+  @moduledoc false
+
+  require OpenApiSpex
+  require Trento.Domain.Enums.ClusterType, as: ClusterType
+
+  alias OpenApiSpex.Schema
+
+  alias TrentoWeb.OpenApi.V1.Schema.{Cluster, Provider, ResourceHealth, Tags}
+
+  defmodule AscsErsClusterNode do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "AscsErsClusterNode",
+      description: "ASCS/ERS Cluster Node",
+      type: :object,
+      properties: %{
+        attributes: %Schema{
+          type: :object,
```

Comment:
As before: is this a `key => value` of type `string => string`? Maybe `additionalProperties` could help.

Reply:
Adding: `additionalProperties: %Schema{type: :string}`